Update lambda function

michlam opened the pull request:
Updates to the lambda function:
- Split original file into two functions, test_process_image and test_retrieve_results
- Includes the updated lambda policy

michlam committed:
Updated lambda_policy.json

- Adjusted policy for group account

michlam committed:
Update lambda_function.py

- Added response waiting, json retrieval, and json formulation for RDS
- Formatting

michlam committed:
fixed process image and retrieve results functions

- Split original lambda into two functions

park-early reviewed (approved):
lgtm (҂◡_◡) ᕤ

will probably have to update the lambda after working out the api gateway stuff

michlam committed:
Updated new fixes

- New changes to process image and retrieve results after api-gateway and uuid changes
